util.c

```diff
@@ -328,6 +328,8 @@ json_t *json_rpc_call(CURL *curl, const char *url,
 
 	curl_easy_setopt(curl, CURLOPT_NOSIGNAL, 1);
 	curl_easy_setopt(curl, CURLOPT_URL, url);
+	if (cert != NULL)
+		curl_easy_setopt(curl, CURLOPT_CAINFO, cert);
```

Comment:
That's fair.  One issue I can see there is that most Windows users won't have `openssl` available.  The approach in this PR allows them to simply add "--cert C:\Path\To\dcrd.cert" to `start_local.bat` without needing any more apps.

However, I believe the approach you propose could be used instead of specifying `--cert` for those that do have `openssl` available.